socorro/cron/base.py

```diff
@@ -156,6 +157,7 @@ class BaseCronApp(RequiredConfig):
     def __init__(self, config, job_information):
         self.config = config
         self.job_information = job_information
+        self.logger = logging.getLogger(__name__ + '.' + self.__class__.__name__)
```

Comment:
I thought about moving this to a lib module because it's kind of icky. So then we could have `get_instance_logger()` and `get_class_logger`, but it'd have to look at the stack to figure out the right class and `__name__` and I decided that's probably not wildly helpful. We could also make a mixin that has the methods, but that seemed unenthusing, too.